documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_2_Identity_and_Trust_Management.md

```diff
@@ -0,0 +1,113 @@
+
+The service instance for a connector is typically a Service including core services, some usage control framework and applications (Data Apps).
+Other components (Broker, DAPS, ...) are represented by their Service (represented by one or multiple containers) running on a comparable platform.
+
+One component always is characterized by the combination of platform and service instances. As an example, this Connector instance is running several data apps. The identity is comprised of the platform, the Connector Core Services and the deployed Data Apps.
+
+![Components SW Stack](./media/SW_Stack_Components_connector_blueprint.png)
+#### _Figure 4.1.2.1: Components of the Software Stack of an IDS Connector_
+
+### Component Identifier
+
+The identity of a combination of platform and service instance is bound to an identifier for the service instance.
+
+* Each component gets a unique identified (C_UID) bound to the service instance.
+* The uniqueness of this identifier is ensured by the Identity Provider.
+* Each UID is mapped to a Connector Instance Key (CIK) pair which is typically used for TLS but possibly also data signing and other identity proofs.
+
+Each Service Instance needs to be mapped to one platform it utilizes:
+* Each platform blueprint gets a unique identifier during the component certification.
+* For a component with Trust level 1, the concrete platform instantiation (running version of this blueprint) does NOT get a UID as well as key and certificate for this platform instance. Instead the connector description solely references the (unique) identifier of the certified blueprint (and the operator needs to be trusted to ensure its correct instantiation).
+* For components with Trust Level 2 or 3, each platform instantiation needs to be uniquely identified with a UID (P_UID). This UID is required to provide a mapping from service instance to platform.
+* Each P_UID is subject of a certificate for the utilized platform key(s) (e.g. AK of a TPM) which enable verification of the platform integrity.
+* One P_UID can map to Platform Instance Keys (PIK) for 1-n physical devices/protected VMs:
+  * If 1 physical device serves as the component platform: 1 P_UID is mapped to 1 PIK.
+  * For distributed platform setups (e.g. with kubernetes): 1 P_UID for the setup maps to n PIKs for the servers in this distributed platform.
+  * If connector is run in one protected VM (e.g. SEV SNP): 1 P_UID is mapped to 1 PIK for this SEV-SNP VM.
+  * If multiple protected VMs (e.g. SEV SNP) form a distributed platform setup: 1 UID for the setup maps to n PIKs for the VMs which comprise this setup.
+
+![Identity mapping for different scenarios](./media/identity_mapping.png)
+#### _Figure 4.1.2.2: Identities for IDS Connector Services and Platforms_
+(Remark: The platforms in the image may always be either physical devices or protected VMs)
+
+### Describing Metadata
```

Comment:
the signed metadata descriptions should in general be available for all Trust Levels as the machine-readable and verifiable version of the IDS certificate (from the IDS Core Component Certification). However, for trust level 1 the descriptions will probably not contain the hash values which could be used to validate the SW stack integrity (since this is not supported by the software) - I have added a few words to make that clearer.